packages/Organisms/Publication/PublicationDownloadPrint.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <ul class="rpl-publication-download-print">
+    <li v-for="(link, index) in links" :key="index">
+      <rpl-document-link class="rpl-publication-download-print__print-item" v-bind="link" />
```

Comment:
probably need to emit an event here on click too - publicationDownload. With the name of the publication as payload

Reply:
Added an event.